src/applications/static-pages/sass/modules/_m-operating-status.scss

```diff
@@ -0,0 +1,107 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.collapsible {
+  color: $color-gray-dark;
+  cursor: pointer;
+  width: 303px;
+  height: 42px;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 16px;
+  padding: 0;
+  margin: 0;
+  background-color: transparent;
+  border-radius: 0;
+
+  @media (max-width: $small-screen) {
+    width: 100%;
+  }
+
+}
+
+.collapsible:hover {
+  color: $color-gray-dark;
+  background-color: transparent;
+}
+
+.collapsible:active {
+  color: $color-gray-dark;
+  background-color: transparent;
+}
+
+.status-label {
+  text-decoration: underline $color-light-blue dotted;
+  text-underline-position: under;
+
+  &:active {
+    pointer-events: none;
+  }
+
+}
+
+.content {
+  width: 303px;
+  display: none;
+  overflow: hidden;
+
+  @media (max-width: $small-screen) {
+    width: 100%;
+  }
+}
+
+.alert-icon-base {
+  width: 20px !important;
+  height: 18px !important;
+  pointer-events: none !important;
+  padding: 12px 11px 12px 12px !important;
+  font-size: unset !important;
+  display: unset !important;
+}
+
+.more-info {
+  margin: 0;
+  padding: 0 12px 12px 12px
+}
+
+.more-icon {
+  pointer-events: none;
+}
+
+.limited {
+  background-color: $color-gold-lightest !important;
+
+  span.status-label:hover {
+    background-color: $color-gold-lighter !important;
+  }
+
+  span.status-label:active {
+    background-color: $color-gold-lighter !important;
+  }
+
+}
+
+.closed {
+  background-color: $color-red-lightest !important;
+
+  span.status-label:hover {
+    background-color: $color-secondary-light !important;
+  }
+
+  span.status-label:active {
+    background-color: $color-secondary-light !important;
+  }
+
+}
+
+.notice {
+  background-color: $color-aqua-lightest !important;
+
+  span.status-label:hover {
+    background-color: $color-primary-alt-light !important;
+  }
+
+  span.status-label:active {
+    background-color: $color-primary-alt-light !important;
+  }
+}
```

Comment:
Please add this file to CODEOWNERS. We should probably put all facility-owned modules in a directory.